### ISSUE TITLE
Feature Request: Support collapsible markdown

### ISSUE THREAD

**Issue by smiller171** [microsoft/vscode#168963]

**Comment by mjbvz** [comment]
@smiller171 @alexr00  Is this for markdown in comments?

**Comment by alexr00** [comment]
@mjbvz  Yes, this is for comments. I assumed that this also didn't work in the markdown preview, but it looks like it does. I've removed you as an assignee. Sorry about the noise!

**Comment by alexr00** [comment]
Note for the future: `<details>` and `<summary>` are part of our html in markdown support.
